src/IntegrationTests/java/uk/gov/hmcts/reform/sscs/callback/UcWriteFinalDecisionIt.java

```diff
@@ -67,7 +67,7 @@ public void callToMidEventPreviewFinalDecisionCallback_willPreviewTheDocumentFor
         final NoticeIssuedTemplateBody parentPayload = (NoticeIssuedTemplateBody) capture.getValue().getFormPayload();
         final WriteFinalDecisionTemplateBody payload = parentPayload.getWriteFinalDecisionTemplateBody();
 
-        assertEquals("An Test", parentPayload.getAppellantFullName());
+        assertEquals("AN Test", parentPayload.getAppellantFullName());
```

Comment:
Since you've done it for the lastname in other files, would be better to save this AN Test to a constant as well